sys/include/net/ng_netconf.h

```diff
@@ -119,8 +119,12 @@ typedef enum {
      * @note not all transceivers may support this interrupt
      */
     NETCONF_OPT_TX_END_IRQ,
-    NETCONF_OPT_AUTOCCA,            /**< en/disable to check automatically
-                                         before sending the channel is clear. */
+    NETCONF_OPT_CSMA,               /**< en/disable to check automatically
```

Comment:
@jremmert-phytec-iot
> The differences between this device and the kw2x device is that the kw2x device does not integrate the mechanisms for backoff handling in hardware. Though it integrates an automatic CCA check before sending out a packet.

I don't quite get this part of your comment. What exactly does the kw2x when its integrated CCA detects that the channel is already occupied?